data/decks/sagat.json

```diff
@@ -0,0 +1,189 @@
+{
+	"id": "sagat",
+	"season": 3,
+	"display_name": "Sagat",
+	"cardback": "cardback_defaultexceed.png",
+	"exceed_cost": 3,
+	"on_exceed": {
+		"effect_type": "draw",
+		"amount": 2,
+		"and": {
+			"effect_type": "choice",
+			"choice": [
+				{ "effect_type": "retreat", "amount": 1 },
+				{ "effect_type": "retreat", "amount": 2 },
+				{ "effect_type": "pass" }
+			]
+		}
+	},
+	"ability_effects": [
+		{
+			"timing": "set_strike",
+			"condition": "is_not_critical",
+			"effect_type": "gauge_for_effect",
+			"per_gauge_effect": null,
+			"overall_effect": {
+				"effect_type": "critical",
+				"character_effect": true
+			},
+			"gauge_max": 1
+		},
+		{
+			"timing": "set_strike",
+			"condition": "initiated_strike",
+			"effect_type": "choice",
+			"choice": [
+				{ "effect_type": "reveal_strike", "and": { "effect_type": "speedup", "amount": 1 } },
```

Comment:
can you put the character_effect = true thing together with the speed boost, along with the speed + power boost in the exceed mode? that just makes the reminder popup happen which is helpful